client/my-sites/plans/plan-overview/plan-features/feature.jsx

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Should this file live in its own directory, with its own stylesheet and `.component-name` prefix?